Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -29,12 +29,17 @@
 #import <memory>
 
 // General DWrite helpers
-COREGRAPHICS_EXPORT CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedString);
+#ifdef __cplusplus
+extern "C++" std::wstring _GetUserDefaultLocaleName();
+#endif
```

Comment:
Discussed offline. It's a little sketch still but we're letting it pass in this case.

---
In reply to: [90993784](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90993784) [](ancestors = 90993784,90742313)